charts/kubeovn-operator/templates/deployment.yaml

```diff
@@ -7,17 +7,18 @@ metadata:
   name: kubeovn-operator-controller-manager
   namespace: {{ .Release.Namespace }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      {{- include "kubeovn-operator.labels" . | nindent 6 }}
+      kubectl.kubernetes.io/default-container: manager
```

Comment:
Can we use something else? This is a well-known one that should be placed in the annotations.

https://kubernetes.io/docs/reference/labels-annotations-taints/#kubectl-kubernetes-io-default-container